.github/workflows/ci.yml

```diff
@@ -17,6 +17,22 @@ on:
 
 
 jobs:
+  docs-GH:
+    name: Build Docs
+    needs: initial_check
+    if: needs.initial_check.outputs.should_release == 'true'
```

Comment:
Why not always run this doc test?

Reply:
Good point. It's a relic from copying the job from CI. I will remove this.